index.ts

```diff
@@ -81,7 +82,7 @@ export function commit_entry(entryType: string, entryContent: string): string {
   return handleSerialization(jsonEncodedParams, (e: u32): u32 => env.hc_commit_entry(e));
 }
 
-export function get_entry(hash: string): string {
+export function get_entry(hash: hash): string {
```

Comment:
I'm feeling on the fence about the addition of `hash` type here...

What's your thinking?

Note one extra thing, in the hc-rust source code, they've mostly refactored out the word `hash` in favour or `address`. Since they went pretty far into this ContentAddressableStorage thing. 

So, if we are going to keep it, we might also want to consider calling it that

Reply:
The reason I added it it was from a chat with Dave Meister a couple of weeks ago. At that time the plan was for the types to be FaasLang types (https://github.com/faaslang/faaslang#types) plus an extra holochain specific type for hash. It is also used in the rust app spec. 

So it sounds like we might be moving away from this? I will change it back to string for now to be more generic